openpype/hosts/houdini/api/lib.py

```diff
@@ -747,3 +752,32 @@ def get_camera_from_container(container):
 
     assert len(cameras) == 1, "Camera instance must have only one camera"
     return cameras[0]
+
+
+def validate_job_path():
```

Comment:
We  might want to allow to pass settings document directly for optimization purposes if you have the settings `dict` available already?

Reply:
> if you have the settings dict available already?

I'm afraid it's not, but why not making it available!